spec/alter_table_statement_spec.cr

```diff
@@ -1,55 +1,44 @@
 require "./spec_helper"
 
 describe LuckyMigrator::AlterTableStatement do
-  it "can alter tables" do
+  it "can alter tables with defaults, indices and options" do
     built = LuckyMigrator::AlterTableStatement.new(:users).build do
-      add name : String
-      add age : Int32
-      add completed : Bool
-      add joined_at : Time
-      add amount_paid : Float
-      add email : String?
-      remove :old_field
-    end
-
-    built.statements.first.should eq <<-SQL
-    ALTER TABLE users
-      ADD name text NOT NULL,
-      ADD age int NOT NULL,
-      ADD completed boolean NOT NULL,
-      ADD joined_at timestamptz NOT NULL,
-      ADD amount_paid decimal NOT NULL,
-      ADD email text,
-      DROP old_field
-    SQL
-  end
-
-  it "sets default values and indices" do
-    built = LuckyMigrator::AlterTableStatement.new(:users).build do
-      add name : String, default: "name"
-      add email : String?, default: "optional"
+      add name : String?
+      add email : String, default: "user@lucky.com", fill_existing_with: "noreply@lucky.com"
```

Comment:
I think that the macro should raise if you try to use default *and* fill_existing_with. If you have a default then I don't think `fill_existing_with` makes sense.

Reply:
Just making sure you saw this ^